bot/exts/moderation/voice_gate.py

```diff
@@ -75,7 +81,7 @@ async def _delete_ping(self, member_id: int) -> None:
             log.trace(f"Voice gate reminder message for user {member_id} was already removed")
 
     @redis_cache.atomic_transaction
-    async def _ping_newcomer(self, member: discord.Member) -> bool:
+    async def _ping_newcomer(self, member: discord.Member) -> tuple:
```

Comment:
Docstring needs to be updated to reflect the new return type, detailing what the second element represents.